upstairs/src/lib.rs

```diff
@@ -1628,6 +1787,7 @@ impl Downstairs {
             for id in kvec.iter() {
                 assert!(*id <= ds_id);
                 let wc = self.state_count(*id).unwrap();
+                // ZZZ
```

Comment:
?

Reply:
Note to self cruft left over.  Gone in a663fa3